src/pages/About/About.js

```diff
@@ -137,6 +136,12 @@ const About: ComponentType<Props> = ({ }: Props) => {
                 considered complete. Data for recent days are constantly being revised as more information becomes
                 available.
             </p>
+            <p className={'govuk-body govuk-body govuk-!-margin-bottom-8'}>
+                The new cases reported in the most recent daily update are shown by the black section of the bars.  The lower                     sections of the bars represent cases previously reported.  This shows how the new cases reported relate to                         specimens taken over the last few days, with the occasional tests coming in with longer delays.
+            </p>
+            <p className={'govuk-body govuk-body govuk-!-margin-bottom-8'}>
```

Comment:
Correction required:
-  Repetitive value of `govuk-body` for the `className` attribute.